### PR TITLE
feat/terraform_demo

### PR DESCRIPTION
**Pull request by maxneuvians**
This PR adds terraform for a demo of the lambda action to be used for demoing purposes

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 18, 16, 14, 18-bullseye, 16-bullseye, 14-bullseye, 18-buster, 16-buster, 14-buster
+ARG VARIANT=16-bullseye
```
Comment (patheard) [inline .devcontainer/Dockerfile:2]: Not a blocker in any sense, but you can get away without a Dockerfile for this devcontainer:
https://github.com/patheard/aws-lambda-zipper/blob/main/.devcontainer/devcontainer.json

File: .devcontainer/base.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 18-bullseye, 16-bullseye, 14-bullseye, 18-buster, 16-buster, 14-buster
+ARG VARIANT=16-bullseye
```
Comment (patheard) [inline .devcontainer/base.Dockerfile:2]: Why the second devcontainer `base.Dockerfile`?

Comment (maxneuvians) [reply .devcontainer/base.Dockerfile:2]: VSCode put it there automagically 🧙 - we can probably eventually stream line them

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (maxneuvians) [reply .devcontainer/Dockerfile:2]: Terragrunt is not a feature in those I think

**Review by patheard — APPROVED** [review]
Looks good - just had a few non-blocking comments.

**Pull request merged**